.github/workflows/azure-static-web-apps-black-water-0eaf5100f.yml

```diff
@@ -34,6 +34,27 @@ jobs:
           api_location: "" # Api source code path - optional
           output_location: "/dist" # Built app content directory - optional
           ###### End of Repository/Build Configurations ######
+    deploy_misc:

```

Comment:
[This article](https://docs.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Clinux) says,

> To use Azure PowerShell or Azure CLI in a GitHub Actions workflow, you need to first log in with the [Azure login](https://github.com/marketplace/actions/azure-login) action.

Can you see if we'll need that?